[CPAAS-2675] Improve docs on SAML FSSO claim names (#763)
chili-publish/grafx-documentation: docs/CHILI-GraFx/guides/setup-fsso/saml/index.md

## 1. Gather necessary info

> Our Client Success team will provide you with the following details:

Reviewer: Maybe this should be more direct in telling customers to submit a ticket

```suggestion
Contact our Client Success team via a support ticket to obtain the following details:
```

| `email` or `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress`   | Required        | The email address of the user                                                                                                    |
| `given_name` or `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname` | Required        | The given name of the user                                                                                                       |
| `family_name` or `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname`  | Required        | The family name of the user                                                                                                      |
| `https://chili-publish.com/CGXGroups`                                             | Optional        | A list of UUIDs of the CHILI GraFx groups the user should be in. <br/> This is only required if you use group-based permissions. |

## 3. CHILI GraFx Configuration

Next CHILI publish needs to configure a few things on the CHILI GraFx side, so your users get redirected to your IDP when logging into CHILI GraFx.

> Please provide us with the following metadata:

Reviewer: ```suggestion
Once you've created a SAML application in your IDP, provide us the following metadata so we can configure things to redirect users to your IDP when logging into CHILI GraFx:
```

| **Name**                 | **Description**                                                                                                                  |
| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------- |
| Sign In URL              | The URL of your IDP that CHILI GraFx should redirect to during sign-in                                                           |
| X509 Signing Certificate | The certificate used by your IDP to sign SAML responses. This is used by CHILI GraFx to verify the authenticity of the responses |

Alternatively, you can provide the **SAML metadata URL**.  

> Above values can be derived from it.

Reviewer: ```suggestion
Alternatively, you can provide the **SAML metadata URL**, the above values can be derived from that.
```
